app/assets/stylesheets/home.css.scss

```diff
@@ -0,0 +1,30 @@
+.text-with-side-image {
+  @include outer-container;
+  @include span-columns(12);
+  .side-image {
+    @include span-columns(3 of 12);
+    border: outset;
+    border-width: thick;
+    margin-right: 117px;
+    margin-top: 100px;
+    .block-text {
```

Comment:
Rule declaration should be preceded by an empty line<br>Selector should have depth of applicability no greater than 2, but was 3